Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,19 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+.app-container {
+  /* max-width: 800px; */
+  margin: auto;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.app-container a {
+  color: black;
+  text-decoration: none;
+}
+
+.app-container a:last-child {
```

Comment:
Great commit messages. Nice use of last-child here, I haven't seen anyone else use it in this way to change the link color in the content.